### ISSUE TITLE
[New Feature] config "Normal" ui

### ISSUE THREAD

**Issue by ofirgall** [phaazon/mind.nvim#18]
Be able to configure the background "Normal" of the mind tree

**Comment by phaazon** [comment]
Yep, that could be fun. The highlights system is already present but doesn’t account for changing the normal color of a buffer. I guess it shouldn’t be that hard to add that.

**Comment by ofirgall** [comment]
Any news about that @phaazon ?

**Comment by phaazon** [comment]
Nope, I haven’t started working on this.

**Comment by aspeddro** [comment]
About this feature, we should provide a field in setup like:

```lua
setup{
  highlights = {
   Normal = {fg = "#000000", bg = "#ffffff"},
   StatusLine = {link = 'StatusLine'}
  }
}
```

The defaults highlights is `link` to equivalent

Can I go ahead with a PR?
